### ISSUE TITLE
initialization functions for sparse tensor

### ISSUE THREAD

**Issue by weiyangfb** [pytorch/pytorch#12310]
Add initialization functions for sparse tensor:

[ ] `torch.nn.init.uniform_`
[ ] `torch.nn.init.normal_`
[ ] `torch.nn.init.constant_`
[ ] `torch.nn.init.xavier_uniform_`
[ ] `torch.nn.init.xavier_normal_`
[ ] `torch.nn.init.kaiming_uniform_`
[ ] `torch.nn.init.kaiming_normal_`

**Comment by SsnL** [comment]
I don't think these makes sense... Aren't they making tensors dense?

**Comment by weiyangfb** [comment]
you're right.. how about to add specialized initialization functions that only init for nnz locations?

**Comment by SsnL** [comment]
They are not standardized and should absolutely be implemented by users themselves.
